ports/lpc55/board_flash.c

```diff
@@ -23,31 +23,94 @@
  */
 
 #include "board_api.h"
+#include "boards.h"
+#include "fsl_iap.h"
+#include "fsl_iap_ffr.h"
 #include "tusb.h" // for logging
 
+
+// FLASH
+#define NO_CACHE        0xffffffff
+
+#define FLASH_PAGE_SIZE 512
+#define FILESYSTEM_BLOCK_SIZE 256
+
+static flash_config_t _flash_config;
+static uint32_t _flash_page_addr = NO_CACHE;
+static uint8_t  _flash_cache[FLASH_PAGE_SIZE] __attribute__((aligned(4)));
+
 //--------------------------------------------------------------------+
 //
 //--------------------------------------------------------------------+
 void board_flash_init(void)
 {
-
+	if (FLASH_Init(&_flash_config) == kStatus_Success) {
```

Comment:
It is minor, if possible please keep the tab size = 2 spaces.

Reply:
I will update to be more consistent.